src/mds/heartbeat/heartbeat_manager.cpp

```diff
@@ -100,13 +101,41 @@ void HeartbeatManager::UpdateChunkServerDiskStatus(
     const ChunkServerHeartbeatRequest &request) {
     // update ChunkServerState status (disk status)
     ChunkServerState state;
-    if (request.diskstate().errtype() != 0) {
+    
+    switch (request.diskstate().errtype())
```

Comment:
Note that the code style should be consistent with the code repository

Reply:
ok.